### DISCUSSION TITLE
Problem with datetime

### DISCUSSION THREAD

**Discussion by shirshak55** [prisma/prisma#6885] [Q&A]
Hi, I am facing issue with datetime. In JS timestamp must be in milliseconds. 

In general timestamps are in seconds. But Prisma is storing timestamps 
in seconds which is fine. But when I use findMany it doesn't multiply it by 
1000, so all the date time comes as 1970-xx-xx

Am I doing something wrong because I expect Prisma should give
correct date value when I call findMany?

Database: sqlite
Prisma Client Version: ^2.22.0-dev.48

**Comment by 2color** [comment]
Hey @shirshak55,

Perhaps you could share a more specific example of what you are trying to do?

**Reply by shirshak55** [reply]
@2color just save any date time and retrieve it ? There may be problem with encoding and decoding the date
